io.welle.welle-gui.yml

```diff
@@ -0,0 +1,92 @@
+app-id: io.welle.welle-gui
+runtime: org.kde.Platform
+runtime-version: '6.8'
+sdk: org.kde.Sdk
+command: welle-io
+finish-args:
+  - '--share=network'
+  - '--share=ipc'
+  - '--filesystem=xdg-documents'
+  - '--socket=pulseaudio'
+  - '--socket=wayland'
+  - '--socket=fallback-x11'
+  - '--device=dri'
+  - '--device=all'
+
+modules:
+  - shared-modules/linux-audio/fftw3f.json
+  
+  - name: libfaad
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: 'https://salsa.debian.org/multimedia-team/faad2.git'
+        tag: debian/2.11.2-1
+
+  - name: libusb
+    buildsystem: autotools
+    config-opts:
+      - '--disable-udev'
+    sources:
+      - type: git
+        url: 'https://github.com/libusb/libusb.git'
+        tag: v1.0.27
+
+  - name: librtlsdr
+    buildsystem: cmake-ninja
+    config-opts:
+      - '-Wno-dev'
+      - '-DDETACH_KERNEL_DRIVER=ON'
+    sources:
+      - type: git
+        url: 'https://gitea.osmocom.org/sdr/rtl-sdr.git'
+        tag: v2.0.2
+
+  - name: libairspy
+    buildsystem: cmake-ninja
+    config-opts:
+      - '-Wno-dev'
+    sources:
+      - type: git
+        url: 'https://github.com/airspy/airspyone_host.git'
+        tag: v1.0.10
+
+  - name: SoapySDR
+    buildsystem: cmake-ninja
+    config-opts:
+      - '-DCMAKE_BUILD_TYPE=Release'
+      - '-Wno-dev'
+    sources:
+      - type: git
+        url: 'https://github.com/pothosware/SoapySDR.git'
+        tag: soapy-sdr-0.8.1
+
+  - name: welle.io
+    buildsystem: qmake
+    build-commands:
+      # Copy welle.io binary
+      - install -Dm755 src/welle-gui/welle-io -t /app/bin
+
+      # Copy icons
+      - install -Dm644 src/welle-gui/icons/16x16/welle-io.png /app/share/icons/hicolor/16x16/apps/io.welle.welle-gui.png
+      - install -Dm644 src/welle-gui/icons/32x32/welle-io.png /app/share/icons/hicolor/32x32/apps/io.welle.welle-gui.png
+      - install -Dm644 src/welle-gui/icons/48x48/welle-io.png /app/share/icons/hicolor/48x48/apps/io.welle.welle-gui.png
+      - install -Dm644 src/welle-gui/icons/128x128/welle-io.png /app/share/icons/hicolor/128x128/apps/io.welle.welle-gui.png
+      - install -Dm644 src/welle-gui/icons/256x256/welle-io.png /app/share/icons/hicolor/256x256/apps/io.welle.welle-gui.png
+      - install -Dm644 io.welle.welle-gui.desktop -t /app/share/applications
+
+      # Add current build date into appdata.xml
+      - sed -i 's/date="1970-01-01"/'date=\""$(date +%Y-%m-%d)"\"'/g' io.welle.welle-gui.appdata.xml
+      # Add app version and git hash into appdata.xml
+      - sed -i 's/version="0.0.0"/'version=\""$(cat src/welle-gui/_current_version)-$(git rev-parse --short HEAD)"\"'/g' io.welle.welle-gui.appdata.xml
+      # Finally copy the appdata.xml
+      - install -Dm644 io.welle.welle-gui.appdata.xml -t /app/share/metainfo
+
+    config-opts:
+      - LIBS+=-L/app/lib
+      - LIBS+=-L/app/lib64
+      - 'QMAKE_LFLAGS+=-Wl,-rpath,/app/lib64'
```

Comment:
/app/lib64 should not exist, use /app/lib

Reply:
Funny, when I was developing the flatpak a few months ago, I needed `/app/lib64`. Anyway, it seems that it is also working without it. Thanks for the hint.

Fixed in https://github.com/AlbrechtL/flathub/commit/c2fd82bc92b1a65e45e151641dc38a503e233e33.